sql/src/main/java/io/crate/operation/reference/information/InformationColumnsExpression.java

```diff
@@ -35,11 +35,8 @@
 
         @Override
         public BytesRef value() {
-            String schema = row.info.ident().tableIdent().schema();
-            if (schema == null) {
-                return DOC_SCHEMA_INFO;
-            }
-            return new BytesRef(schema);
+            assert row.info.ident().tableIdent().name() != null : "table schema can't be null";
```

Comment:
assertion message says the schema can't be null but you're checking against the tableName?